starter-code/apple-pie-starter-code/css/main.css

```diff
@@ -0,0 +1,70 @@
+h1, body, html, main {  
+    margin: 0; 
+} 
+
+html {   
+    font-family: 'Open Sans', sans-serif;   
+    font-size: 18px; 
+}
+
+header {
+    height: 400px;
+    background-size: cover;
+}
+#apple {
+    background-image: url("../images/apple-pie.jpg");  
+}
+header h1 {
+    color: #fff;
+    font-size: 50px;
+    font-weight: 400;
+    text-align: center;
+    padding-top: 170px;
+}
+
+ul, ol {
+    padding-left: 10px;
+}
+
+.main-text {
+    color: #554E4E;
+    padding: 50px 0;
+    margin: 0 10%;
+}
+#ingredients {
+border-top: 1px solid #000000;
+border-bottom: 1px solid #000000;
+}
+#ingredients ul {
+    list-style-type: none;
+}
+#ingredients li:before {
+    content: "-";
+    margin-right: 6px;
+}
+body h2 {
+    color: #000000;
```

Comment:
Al igual que usáis #fff para acortar el #ffffff, siempre y cuando los hexadecimales sean iguales podéis acortarlos a la mitad (3 carácteres en vez de 6). Si vais a acortar uno, acortar todos! O vice versa :) depende de los gustos.